readthedocs/settings/base.py

```diff
@@ -520,6 +520,41 @@ def TEMPLATES(self):
     # Additional binds for the build container
     RTD_DOCKER_ADDITIONAL_BINDS = {}
 
+    # When updating this options,
+    # update the readthedocs/rtd_tests/fixtures/spec/v2/schema.json file as well.
+    RTD_DOCKER_BUILD_SETTINGS = {
+        # Mapping of build.os options to docker image.
+        'os': {
+            'ubuntu-20.04': f'{DOCKER_DEFAULT_IMAGE}:ubuntu20',
+        },
+        # Mapping of build.tools options to specific versions.
+        'tools': {
+            'python': {
+                '2.7': '2.7.18',
+                '3.6': '3.6.15',
+                '3.7': '3.7.12',
+                '3.8': '3.8.12',
+                '3.9': '3.9.7',
+                '3.10': '3.10.0rc2',
+                'pypy3.7': 'pypy3.7-7.3.5',
+                'miniconda3-4.7': 'miniconda3-4.7.12',
```

Comment:
I found this problem while testing this: https://github.com/pyenv/pyenv/issues/2070

I'm not sure what to do yet, one option is to patch pyenv to _not install pip_ after installing miniconda.